### DISCUSSION TITLE
is piped proxy faster or the local scrapper?

### DISCUSSION THREAD

**Discussion by soymadip** [libre-tube/LibreTube#4115] [Q&A] [answered]
There is an option 'disable piped proxy' which will allow libretube to scrap yt locally.

So will it be faster than piped proxy?

**Comment by Bnyro** [comment]
There's no option to scrape locally.

**Reply by soymadip** [reply]
I was talking about this option
![Screenshot_2023-06-26-12-40-15-03_7af1185115278aeb9699bfd161691b7e](https://github.com/libre-tube/LibreTube/assets/84225810/25d80c9b-90fa-4931-9169-06cea73d5344)

**Reply by soymadip** [reply]
The last one

**Reply by Bnyro** [reply]
"Load videos and images"

**Reply by soymadip** [reply]
So it doesn't effect video loading speed?

**Reply by Bnyro** [reply, marked as answer]
It does improve the speed when your VPN is faster than the proxy, but it's still not a local scrapper.

**Reply by soymadip** [reply]
Oh, ok
